### DISCUSSION TITLE
Welcome to YamatoBot Discussions!

### DISCUSSION THREAD

**Discussion by matheussilvacydevs** [matheussilvacydevs/YamatoBot#10] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ManavMalhotra** [comment]
Hey, I just deploy your bot in my local system. I am facing some problems like ramlan.js not found.

Is your bot being not ready yet?

**Reply by matheussilvacydevs** [reply]
Hi bro. Ele está sim. Deve ter sido na hora da manutenção dos sticker. Use o Itachi por enquanto vou privar ele. Obrigado não sabia que estava dando este erro.

**Comment by matheussilvacydevs** [comment]
> ?

Hi bro. My bot, is updating. Is latter, for more oje Mouth ok? My friend? For more question. Speak for me ok? :)

**Reply by ManavMalhotra** [reply]
I am also working on building userbot for whatsapp, I need some help related to converting replied image to sticker or making meme using some commands
